front/app/modules/commercial/representativeness/admin/hooks/createRefDataSubscription.ts

```diff
@@ -0,0 +1,229 @@
+// services
```

Comment:
This logic used to be i n`useReferenceData`, but it became quite complex with the birthyear field. I refactored it a bit and made it work better with TypeScript too